packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -27,27 +26,49 @@ headingLevels.forEach((headingLevel) => test(`should render the correct heading
     await expect(component.locator('h2.c-modal-heading')).toContainText(props.heading);
 }));
 
-test.describe('`When the Pie Modal is closed`', () => {
-    test.skip('should dispatch event `pie-modal-close`', async ({ mount }) => {
-        const messages: string[] = [];
-        const component = await mount(
-            PieModal,
-            {
-                props: {
-                    isOpen: true,
+test.describe('`Pie Modal is closed`', () => {
```

Comment:
I'm still not fully convinced with these tests so will probably need to spend some time reading up on playwright. Ideally I want to be able to read the name of the event in the expects and not relie on the `1`